packages/playwright-test/bundles/utils/package-lock.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "utils-bundle",
   "version": "0.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

Comment:
this is not what we want I think, lets stay on lockfileversion 2 for now?

Reply:
Sure, reverted the lockfile version.